src/lib/y2network/wicked/interfaces_reader.rb

```diff
@@ -51,6 +51,7 @@ class InterfacesReader
       def config
         return @config if @config
 
+        Hwinfo.reset
```

Comment:
Would we like to ensure that it is read always or not?, we could reset it outside in case it was too much responsibility for the method, but then it could be too much responsibility to others..